README.md

```diff
@@ -89,23 +136,37 @@ Accept: application/fs.api+json
 
 {
   "data": {
-    "action": "Move",
+    "action": "move",
     "parameters": {
-      "destination": "/dogs/"
-    }
+      "destination": "dogs/",
+      "flags": [ "force" ]
+    },
   }
 }
 ```
+
 **[⬆ back to top](#table-of-contents)**
 
 # Response
 
 Similar to the request syntax, every response WILL have a JSON object at the root of the response when it contains data, which defines the response's "top level".
```

Comment:
Actually, considering it has exceptions it should probably be SHOULD: https://tools.ietf.org/html/rfc2119#section-1